src/jsMain/js/simulator/windows/ShaderEditorWindow/ShaderEditorWindow.jsx

```diff
@@ -89,6 +97,21 @@ const ShowEditorWindow = (props) => {
     console.log(`previewShow!`);
```

Comment:
Do we really want to be leaving `console.log`s here for production?

Reply:
This PR is pretty far from production. Really more looking for help on getting rid of the tons of unintelligible react errors that happen all the time.